core/scss/utilities/functions/modular-scale/_get-modular-base.scss

```diff
@@ -0,0 +1,15 @@
+@charset "UTF-8";
+
+//
+// get-modular-base($breakpoint)
+//
+// Get a value out of the modular-spacing-base map
+// Returns the pixel value of the $modular-spacing-base variable at a breakpoint.
+//
+// $breakpoint - One of `xs`, `sm`, `md`, `lg`, `xl`, `2xl`
+//
+// Style guide: Functions.ModularScale.get-modular-base
+//
+@function get-modular-base($breakpoint: 'xs') {
```

Comment:
perhaps we call this `get-modular-spacing-base` for a slightly better namespace.